docs/project/code_review.md

```diff
@@ -421,36 +419,6 @@ author can always come back to you if they have questions, and we can always
 revert changes if the resolution for some reason diverges wildly from your
 expectations.
 
-## Merging pull requests
```

Comment:
I think that this had originally been here since it's part of the review (ending it, following approval). Maybe just add a cross-reference link?

Reply:
I should note too, a lot of the text refers to "author" and "reviewer", which seems to lack context when moved to the other doc.